tests/events_test.go

```diff
@@ -26,7 +26,7 @@ const (
 	diskName   = "disk0"
 )
 
-var _ = Describe("[Serial] K8s IO events", func() {
+var _ = Describe("[Serial][sig-storage]K8s IO events", func() {
```

Comment:
This one is very strange. It checks "events" but through storage entities.

If it really belongs to storage, it probably should move to the `storage` folder (although not a must in this PR).

Reply:
@mhenriks can you confirm that? Thanks!

Reply:
@awels can you have a look here?

Reply:
Yes it does look like it should belong in storage.